src/components/NetworkListTable.tsx

```diff
@@ -45,7 +52,39 @@ const NetworkListTable: FC<Props> = ({ onFailure, devices, instance }) => {
     },
   ];
 
-  const networksRows = Object.entries(devices ?? {})
+  const lxdNetworks = Object.entries(devices ?? {})
+    .map(([_, networkDevice]) => {
+      if (networkDevice.type !== "nic") {
+        return null;
+      }
+
+      const network = networks.find(
+        (item) =>
+          item.name === networkDevice.network ||
+          item.name === networkDevice.parent,
+      );
+
+      if (!network) {
+        return null;
+      }
+
+      return network;
+    })
+    .filter((network) => network !== null);
+
+  const shouldDisplayAclsColumn = lxdNetworks.some(
+    (t) => t.config["security.acls"],
+  );
```

Comment:
We should check if any network device has an acl or any network attached to the instance has an acl here. Currently checking  if any network has an acl, that could also be one that is not attached. Or we could have no network with acls, but one instance device having an acl.